### PR TITLE
Dropdown: Adding and trim CSS var

### PR DESCRIPTION
**Pull request by louismaximepiton**
Not sure about the `border-top` change since it links `.dropdown-divider` and the dropdown `border`. -> See https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1423#discussion_r953653270.

### Checks

Please make sure that there is no difference between [deploy](https://deploy-preview-1423--boosted.netlify.app/docs/5.2/components/dropdowns/) and [main](https://main--boosted.netlify.app/docs/5.2/components/dropdowns/).

### CODE REVIEW

File: scss/_dropdown.scss [hunk 1]
```diff
@@ -54,8 +54,9 @@
 // The dropdown menu
 .dropdown-menu {
   // scss-docs-start dropdown-css-vars
+  --#{$prefix}dropdown-zindex: #{$zindex-dropdown}; // Boosted mod
```
Comment (louismaximepiton) [inline scss/_dropdown.scss:57]: This change might be on Bs side too

File: scss/_dropdown.scss [hunk 2]
```diff
@@ -9,14 +9,14 @@
 }
 
 .dropdown-toggle {
+  // Boosted mod
   // scss-docs-start dropdown-toggle-css-vars
-  --#{$prefix}dropdown-toggle-padding-x: #{$dropdown-padding-x}; // Boosted mod
```
Comment (julien-deramond) [inline scss/_dropdown.scss:13]: Don't you think it is important that the `// Boosted mod` comments appear in the docs?
And then that some should be added as well lines 29 to 33?

File: scss/_dropdown.scss (hunk 1 above)
Comment (julien-deramond) [reply scss/_dropdown.scss:57]: Yes, IMO it could be integrated in Bootstrap.

File: scss/_dropdown.scss [hunk 3]
```diff
@@ -195,7 +196,7 @@
   height: 0;
   margin: var(--#{$prefix}dropdown-divider-margin-y) 0;
   overflow: hidden;
-  border-top: $border-width solid var(--#{$prefix}dropdown-divider-bg); // Boosted mod
+  border-top: var(--#{$prefix}dropdown-border-width) solid var(--#{$prefix}dropdown-divider-bg); // Boosted mod
```
Comment (julien-deramond) [inline scss/_dropdown.scss:199]: Atm, we can consider that they are stronly linked together.

File: scss/_dropdown.scss (hunk 2 above)
Comment (louismaximepiton) [reply scss/_dropdown.scss:13]: Ah yes you're right, I totally forgot about the doc!

File: scss/_dropdown.scss (hunk 1 above)
Comment (louismaximepiton) [reply scss/_dropdown.scss:57]: Done in https://github.com/twbs/bootstrap/pull/37021

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**